### ISSUE TITLE
Task - Manage controlled vocabulary terms - Add biological relationships section

### ISSUE THREAD

**Issue by proceps** [SpeciesFileGroup/taxonworks#4069]

**Comment by mjy** [comment]
http://127.0.0.1:3000/tasks/biological_relationships/composer

**Comment by proceps** [comment]
Yes, we do, at the same time we have BiocurationGroup and BiocurationClass both in the controlled vocabulary and can manage everything in a single place. There is a direct clink also to manage biocuration classes and groups, but I cannot do the same for biological relationship, I need a second interface the manage related data.

**Comment by mjy** [comment]
Biocuration Group and Class are subclasses of CVTs, BiologicalRelationships are not.  

@jlpereira  Let's add a link to BiologicalRelationships in the top right, so that we can navigate from CVT management to the existing task.

"Everything in one place", when things are different, like the BR manager (with properties differences etc), get's ugly over time.  

The other reason not to merge is that well thought out biological relationships don't require constant reworking and editing, they are setup, and largely ignored.
